ts/a11y/semantic-enrich.ts

```diff
@@ -240,12 +240,17 @@ export function EnrichedMathDocumentMixin<N, T, D, B extends MathDocumentConstru
     public static OPTIONS: OptionList = {
       ...BaseDocument.OPTIONS,
       enableEnrichment: true,
-      enrichSpeech: 'none',                   // or 'shallow', or 'deep'
```

Comment:
This the `enrichSpeech` option has been moved to the `sre` block, but `enrichSpeech` is [documented](https://docs.mathjax.org/en/latest/options/accessibility.html?highlight=enrichSpeech#the-configuration-block), it would be a breaking change to remove it.  For this version, it would be good to transfer it to `sre` if it is set in the user options.  Something like

``` javascript
      //
      //  For now, move enrichSpeech to sre.speech (will remove enrichSpeech in a later version)
      //
      const options = args[2] || {};
      if (options.enrichSpeech) {
        if (!options.sre) {
          options.sre = {};
        }
        options.sre.speech = options.enrichSpeech;
        delete options.enrichSpeech;
      }
```

before the `super(..args)` should do it.